packages/aws-lambda/lambdas/README.md

```diff
@@ -28,7 +28,7 @@ The AWS Lambda demo is comprised of all Lambda functions in this folder prefixed
 
 ### Initial Setup
```

Comment:
This is out of scope for this PR, but to be honest, I think we should delete everything about the Lambda demo. It has been moved over to the Instana demo repository ages ago and we will never need to deploy the Lambda demo from the sources in this repo, ever again.

Reply:
I can remove the lambda demo as part of the v3 branch 👍